concent_api/api-test.py

```diff
@@ -66,31 +73,28 @@ def create_reject_data(task_id):
     assert task_id >= 0
     assert isinstance(task_id, int)
 
-    current_time = int(datetime.datetime.now().timestamp())
-    message_cannot_compute_task = MessageCannotComputeTask(
-        timestamp = current_time,
-        reason = 'provider-quit',
-        task_id = task_id,
-    )
+    cannot_compute_task = CannotComputeTask()
+    cannot_compute_task.task_to_compute = TaskToCompute()
+    cannot_compute_task.task_to_compute.compute_task_def = ComputeTaskDef()
+    cannot_compute_task.task_to_compute.compute_task_def['task_id'] = task_id
+    cannot_compute_task.reason = CannotComputeTask.REASON.WrongKey
 
-    message_cannot_compute_task_dumped = dump(
-        message_cannot_compute_task,
-        CONCENT_PRIVATE_KEY,
-        CONCENT_PUBLIC_KEY,
-    )
+    serialized_cannot_compute_task      = dump(cannot_compute_task,             PROVIDER_PRIVATE_KEY,   REQUESTOR_PUBLIC_KEY)
+    deserialized_cannot_compute_task    = load(serialized_cannot_compute_task,  REQUESTOR_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY, check_time=False)
 
-    return MessageRejectReportComputedTask(
-        timestamp = current_time,
-        reason = 'cannot-compute-task',
-        message_cannot_compute_task = message_cannot_compute_task_dumped,
-    )
+    reject_report_computed_task = RejectReportComputedTask()
+    reject_report_computed_task.cannot_compute_task = deserialized_cannot_compute_task
+    reject_report_computed_task.reason              = RejectReportComputedTask.Reason.TASK_TIME_LIMIT_EXCEEDED
+    reject_report_computed_task.task_to_compute     = None
+    reject_report_computed_task.task_failure        = None
 
+    return reject_report_computed_task
 
-def print_golem_message(message, indent = 4):
-    assert isinstance(message, Message)
 
+def print_golem_message(message, PRIVATE_KEY, PUBLIC_KEY, indent = 4):
```

Comment:
This version of `print_golem_message()` is now out of sync with the one from `api-integration-test.py`. Never do that. This is asking for problems.

Remove the definition from one of the scripts and import it from the other. Or better, move the function to a separate script.